src/v2/BaseLoginRouter.js

```diff
@@ -29,12 +29,16 @@ import uiSchemaTransformer from './ion/uiSchemaTransformer';
 import i18nTransformer from './ion/i18nTransformer';
 import AppState from './models/AppState';
 import idx from 'idx';
+import { OKTA_STATE_TOKEN_KEY } from './view-builder/utils/Constants';
 
 const startLoginFlow = (settings) => {
+  let stateHandle = sessionStorage.getItem(OKTA_STATE_TOKEN_KEY);
+  if (!stateHandle) {
+    stateHandle = settings.get('stateToken');
+  }
```

Comment:
Given the code in `initialize` function: `this.updateStateHandleInStorage`, is this check redundant ?

Reply:
Yeah that's correct it's redundant, i added it earlier, wanted to be extra careful. But now we have it in `initialize` method we can remove it.